### ISSUE TITLE
Weird windows update attempts

### ISSUE THREAD

**Issue by ihor-drachuk** [WereDev/Wu10Man#41]
While Wu10Man was enabled I've got blue window with text smth like "Need to update. You have 1 hour or update now" (very approximately) with 2 buttons: "OK" and "Restart now".

During reboot (just before it) I saw this
![image](https://user-images.githubusercontent.com/23584913/117207959-3e63ef80-adfd-11eb-8909-267f57023c17.png)

Actually I don't think that something has been installed. So, probably, Wu10Man won this time. **But...**

- Now I have folder "$WINDOWS.~BT" with several Gb inside.
- Also some content is here: C:\Windows\SoftwareDistribution\Download

- I see this
![image](https://user-images.githubusercontent.com/23584913/117208357-b5998380-adfd-11eb-9485-8a0f5155f05f.png)

Pressing on "Update and restart" or "Update and shut down" actually don't start any updates, but also they don't disappear.

So, in short: can you help to find solutions for this? (v)
- Possible "Configuring update"
- Undo "Update and restart" items
- Prevent downloading updates to "$WINDOWS.~BT" and "C:\Windows\SoftwareDistribution\Download"

Let me know if I can assist you somehow

**Comment by WereDev** [comment]
Sadly, once an update is in progress, I don't know how to stop it.  Not sure it's even possible.  Windows Update overwrites some file on the fly and then some during shutdown/startup that have been staged.  It's likely possible to find the staging folder and delete those files, but I'm not sure what files it updates on the fly nor how to restore them.  Assuming that you're using System Restore Points, your best bet is probably to let it do the update and then see if there is a System Restore Point that will allow you to revert to the previous point.

**Comment by ihor-drachuk** [comment]
Okay, understand. Thank you for advice!
